packages/stark-ui/src/modules/table/components/table.component.spec.ts

```diff
@@ -1105,4 +1103,36 @@ describe("TableComponent", () => {
 			expect(component.selectChanged.emit).toHaveBeenCalledWith([dummyData[0]]);
 		});
 	});
+
+	describe("async", () => {
+		const dummyData: object[] = [
+			{ id: 1, description: "dummy 1" },
+			{ id: 2, description: "dummy 2" },
+			{
+				id: 3,
+				description: "dummy 3"
+			}
+		];
+		console.log(dummyData);
+
+		beforeEach(() => {
+			hostComponent.columnProperties = [{ name: "id" }, { name: "description" }];
+			hostComponent.dummyData = undefined; // data starts uninitialized
```

Comment:
Maybe it is less code to write if the dummyData is defined as it was before: `public dummyData: object[] = [];` so you don't need to cast to  `<object[]>` every time. You would just need to cast it in this line to `<any>undefined`. What do you think?

Reply:
I updated the PR. 😄